arrow/src/main/java/org/apache/iceberg/arrow/vectorized/VectorizedArrowReader.java

```diff
@@ -347,7 +347,9 @@ public String toString() {
     }
 
     @Override
-    public void setBatchSize(int batchSize) {}
+    public void setBatchSize(int batchSize) {
+
```

Comment:
Do we require the empty line?

Reply:
Yep, we actually did for the Private Constructors as well but we were ignoring that *everywhere*

```
        <module name="RightCurly"> <!-- Java Style Guide: Nonempty blocks: K & R style -->
            <property name="option" value="alone"/>
            <property name="tokens" value="CLASS_DEF, **METHOD_DEF**, CTOR_DEF, LITERAL_FOR, LITERAL_WHILE, STATIC_INIT, INSTANCE_INIT"/>
        </module>
```

Basically Check-style was just broken before (see my comment on this PR) and didn't work correctly on empty blocks.

Reply:
I don't suppose it's possible to make it okay to have the right curly on the line just after the method definition?